dev/com.ibm.ws.messaging.common/resources/com/ibm/ws/sib/processor/CWSIPMessages.nlsprops

```diff
@@ -1298,7 +1298,7 @@ MESSAGE_EXISTS_ERROR_CWSIP0572.useraction=None
 # {1} The system message id
 # {2} The destination name
 # {3} The action taken
-DELIVERY_DELAY_TIME_WARNING_CWSIP0580=CWSIP05800W: A message with ID {0} system message ID {1} was found in destination {2} with a delivery delay exceeding the limit set by the property sib.msgstore.maximumAllowedDeliveryDelayInterval, that action taken was ''{3}''
+DELIVERY_DELAY_TIME_WARNING_CWSIP0580=CWSIP0580W: A message with ID {0} system message ID {1} was found in destination {2} with a delivery delay exceeding the limit set by the property sib.msgstore.maximumAllowedDeliveryDelayInterval, that action taken was ''{3}''
```

Comment:
I see that just the message number was changed, but here are some comments for the message text, explanation, and user action:
Change "A message with ID {0} system message ID {1} was found in destination {2} with a delivery delay exceeding the limit set by the property sib.msgstore.maximumAllowedDeliveryDelayInterval, that action taken was ''{3}'' to "A message with ID {0}  and system message ID {1} was found in destination {2} with a delivery delay exceeding the limit set by the property sib.msgstore.maximumAllowedDeliveryDelayInterval. The action taken was ''{3}'' ".

Also, if there is truly no action for the user to take, might you make the message an informational message instead of a warning message? If it is really a warning message, then you would need a user action that actually has the user do  something.